Sources/Support/Windows/Stringify.cpp

```diff
@@ -53,6 +53,8 @@ char const *Stringify::ExceptionCode(DWORD code) {
     DO_STRINGIFY(EXCEPTION_PRIV_INSTRUCTION)
     DO_STRINGIFY(EXCEPTION_SINGLE_STEP)
     DO_STRINGIFY(EXCEPTION_STACK_OVERFLOW)
+    case 0xE06D7363:
```

Comment:
Somehow I knew that this was not gonna pass code review but still decided to write that shit 😆